Anlab.Mvc/Client/components/PaymentSelection.tsx

```diff
@@ -17,18 +18,36 @@ export class PaymentSelection extends React.Component<IPaymentProps, any> {
     constructor(props) {
         super(props);
         this.state = {
-            error: ""
+            error: "",
+            accountName: null
         };
 
 
     }
     _renderUcAccount = () => {
         if (this.props.payment.clientType === 'uc') {
             return (
-                <Input type="text" label="UC Account" error={this.state.error} value={this.props.payment.account} maxLength={15} onChange={this.handleAccountChange}/>
+                <div>
+                    <Input type="text" label="UC Account" error={this.state.error} value={this.props.payment.account} maxLength={15} onChange={this.handleAccountChange} onBlur={this.lookupAccount} />
+                    {this.state.accountName}
+                </div>
             );
         }
     }
+
+    lookupAccount = (account: string) => {
+        if (this.state.error === "")
+        {
+            fetch(`/financial/${this.props.payment.account}`, { credentials: 'same-origin' })
+                .then(response => response.json())
+                .then(accountName => this.setState({ accountName }));
```

Comment:
@laholstege If it doesn't find it, clear out the account name:
So if I enter 3-BG13DOE, tab off (finds it), then change it to x-BG13DOE it doesn't find it and keeps the previous name.